### ISSUE TITLE
Multi GPU run : EXCEPTION occurred: ParseValue: No value given for ''.

### ISSUE THREAD

**Issue by Eomtaesan** [Microsoft/CNTK#2297]
I use two NVIDIA 1080 Ti GPU by this command
[ mpiexec.exe -np 2 cntk configfile="pt.cntk" parallelTrain = true ]
,and pt.cntk file is this:

--------------------------------cntk file start--------------------------------
command = Train
stderr = "error.txt"
modelPath = "Models/Train.dnn"
parallelTrain = true
deviceId = "auto"

Train = [
	action = "train"
	BrainScriptNetworkBuilder = [

		inputDim = 52:119:1
		outputDim = 11900

		model = Sequential (
			ConvolutionalLayer {192, (3:3), pad = true, bias = true, initBias = 0.5} : ReLU :
			BatchNormalizationLayer {spatialRank = 2} :
			ConvolutionalLayer {192, (3:3), pad = true, bias = true, initBias = 0.5} : ReLU :
			BatchNormalizationLayer {spatialRank = 2} :
			ConvolutionalLayer {1, (1:1), pad = true, bias = true, initBias = 0.5} : ReLU :
			BatchNormalizationLayer {spatialRank = 2} :
			DenseLayer {4096, activation = ReLU, bias = true, initBias = 1} : Dropout :
			LinearLayer {outputDim, bias = true, initBias = 0.5}
		)

		features = Input (inputDim)
		labels = Input (outputDim)
		z = model (features)

		ce = CrossEntropyWithSoftmax (labels, z)
		errs = ClassificationError (labels, z)

		featureNodes = (features)
		labelNodes = (labels)
		criterionNodes = (ce)
		evaluationNodes = (errs)
		outputNodes = (z)
	]
	SGD = [
		epochSize = 0
		minibatchSize = 256
		learningRatesPerSample = 0.0000000001
		minLearningRatePerSample = 0.0000000000000000000000000000000000000000000000000000000001
		maxEpochs = 256
		DropoutRate = 0.5
		numMBsToShowResult = 1
		ParallelTrain = [
			parallelizationMethod  = DataParallelSGD
			parallelizationStartEpoch = 2
			distributedMBReading = true
			syncPerfStats = 10
			dataParallelSGD = [ gradientBits = 2 ]
		]
	]
	reader = [
		readerType = "CNTKTextFormatReader"
		file = "GPW.txt"
		input = [
			features = [
				dim = 6188
				format = "dense"
			]
			labels = [
				dim = 11900
				format = "dense"
			]
		]
	]
]
--------------------------------cntk file end--------------------------------

Error message is this:

--------------------------------error message start--------------------------------
CNTK 2.1 (HEAD 5643a5, Jul 31 2017 03:44:29) on DESKTOP-KKK6IE7 at 2017/09/03 01:11:45

cntk  configfile=pt.cntk  parallelTrain  =  true

[CALL STACK]
    > Microsoft::MSR::CNTK::Matrix<float>::  __autoclassinit2
    - Microsoft::MSR::CNTK::Matrix<float>::  __autoclassinit2 (x5)
    - Microsoft::MSR::CNTK::MatrixBase::  MatrixBase
    - BaseThreadInitThunk
    - RtlUserThreadStart

EXCEPTION occurred: ParseValue: No value given for ''.
CNTK 2.1 (HEAD 5643a5, Jul 31 2017 03:44:29) on DESKTOP-KKK6IE7 at 2017/09/03 01:11:45

cntk  configfile=pt.cntk  parallelTrain  =  true

[CALL STACK]
    > Microsoft::MSR::CNTK::Matrix<float>::  __autoclassinit2
    - Microsoft::MSR::CNTK::Matrix<float>::  __autoclassinit2 (x5)
    - Microsoft::MSR::CNTK::MatrixBase::  MatrixBase
    - BaseThreadInitThunk
    - RtlUserThreadStart

EXCEPTION occurred: ParseValue: No value given for ''.
--------------------------------error message end--------------------------------

I setup "MSMpiSetup"
I do not know why this error occur, what is this error?

**Comment by KeDengMS** [comment]
The error comes from parsing the script for some key=value pair, where the value is missing. Did this script work before? Have you tried on single GPU? Can you include GPW.txt?

**Comment by cha-zhang** [comment]
Close due to lack of response. Please reopen if needed. Thanks!
